src/instances.ts

```diff
@@ -316,7 +316,7 @@ function addAssetHooks(loader: WebpackLoaderContext, instance: TSInstance) {
   // compilation is actually of type webpack.Compilation, but afterProcessAssets
   // only exists in webpack5 and at the time of writing ts-loader is built using webpack4
   const makeAssetsCallback = (compilation: any) => {
-    compilation.hooks.afterProcessAssets.tap('ts-loader', () =>
+    compilation.hooks.processAssets.tap({ name: 'ts-loader', stage: webpack.Compilation.PROCESS_ASSETS_STAGE_ADDITIONAL }, (_assets: webpack.Asset[]) => {
```

Comment:
I'm not sure what we decided in this one? It's the same thing as the fix above. Both works, but one is newer and preferred.
`_assets: webpack.Asset[]` could be removed if you prefer leaving info out rather then keep it for future usage. I usually leave it so you don't have to dig it up when you need it. But I don't have a strong opinion about it.